### PR TITLE
remove inline base64 img uri from avatar styles

### PR DESCRIPTION
**Pull request by akdetrick**
#### Related issues
Fixes https://meetup.atlassian.net/browse/WC-162

#### Description
- removed unused CSS for `avatar--icon`
- removed base64 uri from avatar styles (replaces org badge img with badge svg icon)
- refactored `avatarCharm` placeholder into a mixin that positions an actual `Icon` component
- updated stories for `AvatarMember` to show available sizes

#### Screenshots (if applicable)
screen shot 2018-02-02 at 4 08 58 pm
screen shot 2018-02-02 at 4 09 06 pm

### CODE REVIEW

File: assets/scss/components/_avatar.scss [hunk 1]
```diff
@@ -148,7 +154,7 @@ $avatarIconSizeProps: (height, line-height, width);
 	text-indent: initial;
 	background-color: darken($avatarBg, 6%);
 
-	.svg {
+	.svg--profile {
```
Comment (mmcgahan) [inline assets/scss/components/_avatar.scss]: does this still need to be nested under `.avatar--noPhoto`?

Comment (akdetrick) [reply assets/scss/components/_avatar.scss]: Unfortunately, yes, until we move to CSS Modules. There's a chance we may use an `Icon shape="profile"` outside of an avatar, and disqualifying these rules from `.avatar--noPhoto` could cause side effects.

File: assets/scss/components/_avatar.scss [hunk 2]
```diff
@@ -40,37 +33,50 @@ Class                     | Description
...
 
-	&:after {
-		background-position: center;
-		background-repeat: no-repeat;
-		background-size: 10px;
-		border-radius: 999px;
+	// Use proportionally smaller badge for larger avatars
+	&.avatar--large .svg--avatarBadge,
+	&.avatar--xxlarge .svg--avatarBadge {
+		width: 30%;
```
Comment (mperrotti) [inline assets/scss/components/_avatar.scss]: Is there a reason this is `30%` instead of a more specific value (e.g.: `{n}px`)

Comment (akdetrick) [reply assets/scss/components/_avatar.scss]: Yep - using a `%` relative to the parent makes this work with media-scaling avatars

**Pull request merged**